src/middlewared/middlewared/etc_files/docker.py

```diff
@@ -27,7 +23,7 @@ def nvidia_configuration(middleware):
 
 
 def gpu_configuration(middleware):
-    available_gpu = middleware.call_sync('hardware.available_gpu')
+    available_gpu = middleware.call_sync('device.get_gpus')
```

Comment:
We are not supposed to call them directly, but device.get_info GPU, because the others are private methods, unless strictly necessary